pkg/kubelet/kubelet_pods.go

```diff
@@ -893,13 +895,24 @@ func (kl *Kubelet) getPullSecretsForPod(pod *v1.Pod) []v1.Secret {
 		}
 		secret, err := kl.secretManager.GetSecret(pod.Namespace, secretRef.Name)
 		if err != nil {
-			klog.InfoS("Unable to retrieve pull secret, the image pull may not succeed.", "pod", klog.KObj(pod), "secret", klog.KObj(secret), "err", err)
+			klog.InfoS("Unable to retrieve pull secret, the image pull may not succeed.",
+				"pod", klog.KObj(pod),
+				"secret", klog.KObj(secret),
+				"secretName", secretRef.Name,
+				"err", err)
```

Comment:
adding `secretName` to the log is not needed as `klog.KObj(secret)` already contains the name.